<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SeerCatchpointCreateDialogForm</class>
 <widget class="QDialog" name="SeerCatchpointCreateDialogForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>304</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create a Catchpoint</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="catchpointTypeGroupBox">
     <property name="title">
      <string>Catchpoint Type</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QRadioButton" name="throwRadioButton">
        <property name="text">
         <string>Catch on 'throw'</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">typeButtonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="loadRadioButton">
        <property name="text">
         <string>Catch on library 'load'</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">typeButtonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="rethrowRadioButton">
        <property name="text">
         <string>Catch on 'rethrow'</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">typeButtonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QRadioButton" name="unloadRadioButton">
        <property name="text">
         <string>Catch on library 'unload'</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">typeButtonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="catchRadioButton">
        <property name="text">
         <string>Catch on 'catch'</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">typeButtonGroup</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="catchpointDetailsGroupBox">
     <property name="title">
      <string>Catchpoint Details</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="2">
       <widget class="QCheckBox" name="temporaryCheckBox">
        <property name="text">
         <string>Temporary</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="nameLabel">
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="nameLineEdit">
        <property name="placeholderText">
         <string>Exception/Library name as a regular expression.</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SeerCatchpointCreateDialogForm</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SeerCatchpointCreateDialogForm</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="typeButtonGroup"/>
 </buttongroups>
</ui>
